cloudbuild.yaml

```
@@ -62,21 +70,26 @@ steps:
-    docker tag bazel/base:base_debian9            gcr.io/$PROJECT_ID/base-debian9:latest
-    docker tag bazel/base:base_debian10           gcr.io/$PROJECT_ID/base-debian10:${COMMIT_SHA}
-    docker tag bazel/base:base_debian10           gcr.io/$PROJECT_ID/base-debian10:latest
-    docker tag bazel/base:base_nonroot_debian9    gcr.io/$PROJECT_ID/base:nonroot
-    docker tag bazel/base:base_nonroot_debian9    gcr.io/$PROJECT_ID/base-debian9:nonroot
-    docker tag bazel/base:base_nonroot_debian10   gcr.io/$PROJECT_ID/base-debian10:nonroot
-    docker tag bazel/base:debug_debian9           gcr.io/$PROJECT_ID/base:debug
-    docker tag bazel/base:debug_debian9           gcr.io/$PROJECT_ID/base-debian9:debug
-    docker tag bazel/base:debug_debian10          gcr.io/$PROJECT_ID/base-debian10:debug
-    docker tag bazel/base:debug_nonroot_debian9   gcr.io/$PROJECT_ID/base:debug-nonroot
-    docker tag bazel/base:debug_nonroot_debian9   gcr.io/$PROJECT_ID/base-debian9:debug-nonroot
-    docker tag bazel/base:debug_nonroot_debian10  gcr.io/$PROJECT_ID/base-debian10:debug-nonroot
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base:${COMMIT_SHA}
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base:latest-amd64
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base-debian9:${COMMIT_SHA}
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base-debian9:latest
+    docker tag bazel/base:base_root_amd64_debian10       gcr.io/$PROJECT_ID/base-debian10:${COMMIT_SHA}
+    docker tag bazel/base:base_root_amd64_debian10       gcr.io/$PROJECT_ID/base-debian10:latest
+    docker tag bazel/base:base_nonroot_amd64_debian9     gcr.io/$PROJECT_ID/base:nonroot-amd64
+    docker tag bazel/base:base_nonroot_amd64_debian9     gcr.io/$PROJECT_ID/base-debian9:nonroot
+    docker tag bazel/base:base_nonroot_amd64_debian10    gcr.io/$PROJECT_ID/base-debian10:nonroot
+    docker tag bazel/base:debug_root_amd64_debian9       gcr.io/$PROJECT_ID/base:debug-amd64
+    docker tag bazel/base:debug_root_amd64_debian9       gcr.io/$PROJECT_ID/base-debian9:debug
+    docker tag bazel/base:debug_root_amd64_debian10      gcr.io/$PROJECT_ID/base-debian10:debug
+    docker tag bazel/base:debug_nonroot_amd64_debian9    gcr.io/$PROJECT_ID/base:debug-nonroot-amd64
+    docker tag bazel/base:debug_nonroot_amd64_debian9    gcr.io/$PROJECT_ID/base-debian9:debug-nonroot
+    docker tag bazel/base:debug_nonroot_amd64_debian10   gcr.io/$PROJECT_ID/base-debian10:debug-nonroot
+
+    docker tag bazel/base:base_root_arm64_debian9        gcr.io/$PROJECT_ID/base:latest-arm64
+    docker tag bazel/base:base_root_arm64_debian9        gcr.io/$PROJECT_ID/base:nonroot-arm64
+    docker tag bazel/base:debug_root_arm64_debian9       gcr.io/$PROJECT_ID/base:debug-arm64
+    docker tag bazel/base:debug_root_arm64_debian9       gcr.io/$PROJECT_ID/base:debug-nonroot-arm64
```

```suggestion
    docker tag bazel/base:base_root_arm64_debian9        gcr.io/$PROJECT_ID/base:latest-arm64
    docker tag bazel/base:base_nonroot_arm64_debian9     gcr.io/$PROJECT_ID/base:nonroot-arm64
    docker tag bazel/base:debug_root_arm64_debian9       gcr.io/$PROJECT_ID/base:debug-arm64
    docker tag bazel/base:debug_nonroot_arm64_debian9    gcr.io/$PROJECT_ID/base:debug-nonroot-arm64
```

And likewise, eventually we should also tag these in `-debian9`.

---

Good catch!  re: `-debian9` I think we need a better strategy for scripting the variants, but totally agree.  This is quite tedious to keep straight as-is 😅